src/commands/logging/logger.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2017 Oursky Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import sqlFormatter from 'sql-formatter';
+
+import { LogColorizer } from './coloring';
+
+export type Logger = (logData: any) => string;
```

Comment:
is log data always a `{string: any} | null`?

Reply:
Should be. Let me update the type definition.